src/dispatch/database/revisions/tenant/versions/2023-03-14_b48b245f9c4c.py

```diff
@@ -0,0 +1,54 @@
+"""empty message
+
+Revision ID: b48b245f9c4c
+Revises: 7db13bf5c5d7
+Create Date: 2023-03-14 16:56:11.844768
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = "b48b245f9c4c"
+down_revision = "7db13bf5c5d7"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        "assoc_signal_workflows",
+        sa.Column("signal_id", sa.Integer(), nullable=False),
+        sa.Column("workflow_id", sa.Integer(), nullable=False),
+        sa.ForeignKeyConstraint(["signal_id"], ["signal.id"], ondelete="CASCADE"),
+        sa.ForeignKeyConstraint(["workflow_id"], ["workflow.id"], ondelete="CASCADE"),
+        sa.PrimaryKeyConstraint("signal_id", "workflow_id"),
+    )
+    op.create_table(
+        "assoc_signal_instance_tags",
+        sa.Column("signal_instance_id", postgresql.UUID(as_uuid=True), nullable=False),
+        sa.Column("tag_id", sa.Integer(), nullable=False),
+        sa.ForeignKeyConstraint(["signal_instance_id"], ["signal_instance.id"], ondelete="CASCADE"),
+        sa.ForeignKeyConstraint(["tag_id"], ["tag.id"], ondelete="CASCADE"),
+        sa.PrimaryKeyConstraint("signal_instance_id", "tag_id"),
+    )
+    op.drop_index("ix_entity_search_vector", table_name="entity")
+    op.create_index(
```

Comment:
I need to look into this because i'm not sure if it should be here/how it was introduced.

Reply:
It looks like it's dropping it and re-creating it. Might be an alembic anomaly.